ftplugin/NeogitGitCommandHistory.vim

```diff
@@ -1,15 +0,0 @@
-" Only do this when not done yet for this buffer
-if exists("b:did_ftplugin")
-  finish
-endif
-let b:did_ftplugin = 1
-
-function! NeogitFoldFunction()
-  return getline(v:foldstart)
-endfunction
-
-setlocal foldmethod=manual
-setlocal foldlevel=1
-setlocal fillchars=fold:\ 
-setlocal foldminlines=0
-setlocal foldtext=NeogitFoldFunction()
```

Comment:
The command history still uses folds though, the changes made only apply to the status buffer. So this one needs to stay in for now. Sorry for not stating that more clearly before.

Reply:
Got it, thanks!